storage-operator/deploy/crds/storage_v1alpha1_volume_crd.yaml

```diff
@@ -67,6 +67,56 @@ spec:
           - storageClassName
           type: object
         status:
+          properties:
+            failedContext:
+              properties:
+                errorCode:
+                  enum:
+                  - 0
+                  - 1
+                  - 2
+                  - 3
+                  format: int64
+                  type: integer
+                errorMessage:
+                  type: string
+              type: object
+            pendingContext:
+              properties:
+                jobId:
+                  description: JID of the Salt job in progress.
+                  type: string
+                size:
+                  description: Size of the volume.
+                  type: string
+                step:
+                  description: Name of the current step.
+                  enum:
+                  - PreparationStarted
+                  - PrepareVolume
+                  - GetVolumeSize
```

Comment:
Let's not make this a state.

Reply:
Why?
It's a required step to create a PV from RawBlockDevice, something that I need to start and then poll (like `PrepareVolume`) so it makes sense, no?

Reply:
As seen with you, I'll rework this part and the `GetVolumeSize` step will be removed.